cli/lsp/testing/execution.rs

```diff
@@ -61,12 +61,12 @@ fn as_queue_and_filters(
           if let Some(test) = test_definitions.get_by_id(id) {
             let filter =
               filters.entry(item.text_document.uri.clone()).or_default();
-            if let Some(include) = filter.maybe_include.as_mut() {
+            if let Some(include) = filter.include.as_mut() {
```

Comment:
arbitrary renaming, why?  Our convention is that options are named "maybe_".

Reply:
That's not true, I saw https://github.com/denoland/deno/blob/7d9e8baebb6c2534346e0823e2cc6408ebb8b1eb/cli/lsp/testing/lsp_custom.rs#L90-L99 and plenty of other places. It was attractive to switch here because `exclude` was made non-optional.

In my experience, prefixes like `try_foo()` and `maybe_foo()` are used for _methods_ to distinguish from their counterparts which panic on failure/absence. It's usually not needed for fields and variables thanks to editors, it's sort of Hungarian